src/auth/user-record.ts

```diff
@@ -120,9 +128,13 @@ export abstract class MultiFactorInfo {
    */
   public static initMultiFactorInfo(response: MultiFactorInfoResponse): MultiFactorInfo | null {
     let multiFactorInfo: MultiFactorInfo | null = null;
-    // Only PhoneMultiFactorInfo currently available.
+    // PhoneMultiFactorInfo, TotpMultiFactorInfo currently available.
     try {
-      multiFactorInfo = new PhoneMultiFactorInfo(response);
+      if (response.phoneInfo !== undefined) {
+        multiFactorInfo = new PhoneMultiFactorInfo(response);
+      } else if (response.totpInfo !== undefined) {
+        multiFactorInfo = new TotpMultiFactorInfo(response);
+      }
     } catch (e) {
       // Ignore error.
```

Comment:
nit: can we add an empty else block with comment like - 
Ignore the other SDK unsupported MFA factors to prevent blocking developers using the current SDK.